app.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+
+require('dotenv').config();
+require('./config/db');
+app.use(morgan('dev'));
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use((req, res, next) => {
+    res.header("Access-Control-Allow-Origin", "*");
+    res.header(
+        "Access-Control-Allow-Headers",
+        "Origin, X-Requested-With, Content-Type, Accept, Authorization"
+    );
+    if (req.method === "OPTIONS") {
+        res.header("Access-Control-Allow-Methods", "PUT, POST, PATCH, DELETE, GET");
+        return res.status(200).json({});
+    }
+    next();
+});
+
+app.use('/api/listing', require('./api/routes/listing'));
+app.use('/api', require('./api/routes/contact-us'));
+
```

Comment:
I suggest you add an API call called `/health` to check for the status of the server. It's always good practice to include this in any server.
Google the standards of such a controller.